pallets/ds-accounts/src/lib.rs

```diff
@@ -237,70 +279,6 @@ decl_module! {
             Self::deposit_event(RawEvent::AccountDisabled(who, whom));
```

Comment:
Are we disabling accounts by deleting them completely or just by setting Role to 0? I was under impression that we should use the latter approach, but in the code it is commented out.